config/tf_modules/k8s-service/k8s-service/templates/ingress.yaml

```diff
@@ -24,14 +24,6 @@ metadata:
     nginx.ingress.kubernetes.io/proxy-send-timeout: "3600"
     {{- end }}
 spec:
-  tls:
```

Comment:
Would this have any backwards compatibility consequences?

Reply:
No